.github/workflows/build-pr-artifacts.yml

```diff
@@ -0,0 +1,124 @@
+name: Build Artifacts for PRs
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+env:
+  REPO_NAME_OLD: rudderlabs/develop-rudder-transformer
+  REPO_NAME: rudderstack/develop-rudder-transformer
+  DOCKERHUB_USERNAME: rudderlabs
+
+jobs:
+  build-transformer-image:
+    name: Build Transformer Docker Image - PR
+    runs-on: ubuntu-latest
+
+    # Skip for the release pull requests
+    if: startsWith(github.event.pull_request.head.ref, 'release/') != true && startsWith(github.event.pull_request.head.ref, 'hotfix-release/') != true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - name: Generate Image Version
+        id: gen_img_version
+        shell: bash
+        run: |
+          echo "version=$(echo ${{ github.head_ref }} | tr "/" .)" >> $GITHUB_OUTPUT
```

Comment:
Any sample values of this version ?
As we are removing branch name, does this still be distinct per PR ?

Reply:
The version will be the branch name where the problematic characters (`/`) are replaced (`.`).
Example:
`fix/somefix/1` becomes `fix.somefix.1`
`feat/newfeat` becomes `feat.newfeat`